doc/locale_api_WIP.md

```diff
@@ -0,0 +1,46 @@
+
+Note: ListFoo APIs combine the IDs and the translations. 
+If it's easier to return just the IDs and ask for the translations separately,
+let's use such different API.
+
+### Locales
+
+```
+ListLocales(
+  out a(sss) id_english_native
+)
+
+SetLocale(in s locale_id)
+```
+
+### Keyboards
+
+```
+ListX11Keyboards(
+  out as    ids  # id like "layout(variant)"
+)
+
+SetX11Keyboard(in s kb_id)
+
+X11KeyboardForLocale(
```

Comment:
and same here. Keyboard is linked to language or territory, but not to locale. So we do not have such data.